assets/src/edit-story/components/statusCheck/statusCheck.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useCallback, useEffect } from 'react';
+/**
+ * Internal dependencies
+ */
+import { useAPI } from '../../app';
+
+function StatusCheck() {
+  const {
+    actions: { getStatusCheck },
+  } = useAPI();
+
+  const doStatusCheck = useCallback(() => {
+    getStatusCheck()
+      .catch(() => {
+        // @todo, show error message.
```

Comment:
I think we also need a `.then()`, in case the response is a `200` but the response value is not as expected. So we need to check the response value here.